iothub/service/src/RetryPolicies/IotHubServiceRetryPolicyBase.cs

```diff
@@ -57,7 +57,7 @@ protected TimeSpan UpdateWithJitter(double baseTimeMs)
             lock (_rngLock)
             {
                 // A random double from 95% to 105% of the baseTimeMs
-                jitterMs = _rng.Next(95, 106);
+                jitterMs = _rng.Next(95, 105);
```

Comment:
Remember we decided to change the test case instead of this value?

Reply:
yup, it looks like an old commit got overwritten here. It'll be fixed in the final PR